README.md

```diff
@@ -33,19 +33,33 @@ For more information, see the online manual: [Read The Docs (rpm-ostree)](https:
  - Transactional, background image-based (versioned/checksummed) upgrades
  - OS rollback without affecting user data (`/usr` but not `/etc`, `/var`) via libostree
  - Client-side package layering (and overrides)
- - Easily make your own: `rpm-ostree compose tree`
+ - Easily make your own: `rpm-ostree compose tree` and [CoreOS Assembler](https://github.com/coreos/coreos-assembler)
 
 Projects using rpm-ostree
 --------------------------
 
-[Project Atomic](http://www.projectatomic.io/) is an umbrella project for
-delivering upstream container technologies and combined with a minimized,
-atomically upgradable host system to Fedora, Red Hat Enterprise Linux, and CentOS.
+The default for rpm-ostree is to use [OSTree](https://github.com/ostreedev/ostree).
```

Comment:
The "default" wording here is a bit weird. I'm guessing you're referring to rpm-ostree using only OSTree when not in "hybrid" mode, right? I think for the purposes of this section, something simpler would be nicer, e.g. "At its core, rpm-ostree uses OSTree." ?

Reply:
Eh, just dropped that bit.